src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml.cs

```diff
@@ -73,21 +144,26 @@ private void UserControl_DataContextChanged(object sender, DependencyPropertyCha
             }
         }
 
-        private void PackageReadmeControl_Unloaded(object sender, RoutedEventArgs e)
+        private void PackageReadmeControl_Loaded(object sender, RoutedEventArgs e)
         {
-            if (ReadmeViewModel is not null)
+            NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
             {
-                ThreadHelper.JoinableTaskFactory.Run(async () =>
+                if (!string.IsNullOrWhiteSpace(ReadmeViewModel.ReadmeMarkdown))
                 {
-                    await _markdownPreview.UpdateContentAsync("", ScrollHint.None);
-                });
-                ReadmeViewModel.PropertyChanged -= ReadmeViewModel_PropertyChanged;
-            }
+                    await UpdateMarkdownAsync(ReadmeViewModel.ReadmeMarkdown);
+                }
+            }).PostOnFailure(nameof(PackageReadmeControl), nameof(PackageReadmeControl_Loaded));
```

Comment:
@nkolev92 It seems this extension method that is attached to JoinableTask is forcing folks to `RunAsync` for fire and forget methods when it does nothing but overhead to these paths. That does not seem correct, see" https://github.com/NuGet/NuGet.Client/pull/6190#discussion_r1889485483.